devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -0,0 +1,220 @@
+			tileAttribute("device.alarm", key: "PRIMARY_CONTROL") {
+				attributeState "off", label: 'off', action: 'alarm.siren', icon: "st.alarm.alarm.alarm", backgroundColor: "#ffffff"
+				attributeState "both", label: 'ring!', action: 'alarm.off', icon: "st.alarm.alarm.alarm", backgroundColor: "#0e7507"
+			}
+		}
+		standardTile("off", "device.alarm", inactiveLabel: false, decoration: "flat") {
+			state "default", label: '', action: "alarm.off", icon: "st.secondary.off"
+		}
+		valueTile("tamper", "device.tamper", height: 2, width: 2, decoration: "flat") {
+			state "clear", label: 'tamper clear', backgroundColor: "#ffffff"
+			state "detected", label: 'tampered', backgroundColor: "#ffffff"
+		}
+
+		main "alarm"
+		details(["alarm", "off", "tamper"])
+	}
+}
+
+private getNumberOfSounds() {
+	def numberOfSounds = [
+			"0003": 8 //Aeotec Doorbell 6
+	]
+	return numberOfSounds[zwaveInfo.prod] ?: 1
+}
+
+def installed() {
+	initialize()
+	sendEvent(name: "alarm", value: "off", isStateChange: true, displayed: false)
+	sendEvent(name: "tamper", value: "clear", isStateChange: true, displayed: false)
+}
+
+def updated() {
+	initialize()
+}
+
+def initialize() {
+	if (!childDevices) {
+		addChildren(numberOfSounds)
+	}
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def parse(String description) {
+	def result = null
+	def cmd = zwave.parse(description)
+	if (cmd) {
+		result = zwaveEvent(cmd)
+	}
+	log.debug "Parse returned: ${result.inspect()}"
+	return result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+	createEvent(name: "alarm", value: cmd.value ? "both" : "off")
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.multichannelv3.MultiChannelCmdEncap cmd) {
+	def encapsulatedCommand = cmd.encapsulatedCommand()
+	def endpoint = cmd.sourceEndPoint
+	if (endpoint == state.lastTriggeredSound && encapsulatedCommand != null) {
+		zwaveEvent(encapsulatedCommand)
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd) {
+	def securedEncapsulatedCommand = cmd.securedEncapsulatedCommand()
+	if (securedEncapsulatedCommand) {
+		zwaveEvent(securedEncapsulatedCommand)
+	} else {
+		log.warn "Unable to extract encapsulated cmd from $cmd"
+		createEvent(descriptionText: cmd.toString())
+	}
+}
+
+private onOffCmd(value) {
+	encap(zwave.basicV1.basicSet(value: value))
+}
+
+def on() {
+	resetActiveSound()
+	state.lastTriggeredSound = 1
+	onOffCmd(0xFF)
+}
+
+def off() {
+	state.lastTriggeredSound = 1
+	onOffCmd(0x00)
+}
+
+def strobe() {
+	on()
+}
+
+def siren() {
+	on()
+}
+
+def both() {
+	on()
+}
+
+def ping() {
+	encap(zwave.basicV1.basicGet())
+}
+
+private addChildren(numberOfSounds) {
+	for (def endpoint : 2..numberOfSounds) {
+		try {
+			String childDni = "${device.deviceNetworkId}:$endpoint"
+
+			addChildDevice("Aeotec Child Siren", childDni, device.getHub().getId(), [
```

Comment:
I see you renamed the two files. Don't forget to update the DTH names in the definition blocks and update the name here!

Reply:
Ah, sorry!